runtime/lua/vim/fs.lua

```diff
@@ -388,22 +506,26 @@ function M.normalize(path, opts)
   -- Convert path separator to `/`
   path = path:gsub(os_sep, '/')
```

Comment:
this converts path slashes. so why does `split_windows_path` (called below) support `\` slashes ?

Reply:
Reasoning given above